.github/workflows/pr-tests.yml

```
@@ -0,0 +1,169 @@
+          restore-keys: |
+            ${{ runner.os }}-pip-
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements-dev.txt -c constraints.txt
+
+      # paths-filter outputs: code, docs (always set by dorny/paths-filter@v2)
+      - name: Determine changes
+        id: filter
+        uses: dorny/paths-filter@v2
```

actionlint (1.7.8)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)